module/solutions/helper.js

```diff
@@ -231,11 +494,11 @@ module.exports = class SolutionsHelper {
    * @returns {JSON} - List of solutions based on role and location.
    */
 
-  static forUserRoleAndLocation(bodyData, type, subType = '', pageSize, pageNo, searchText = '') {
+  static forUserRoleAndLocation(bodyData, type, subType = '', programId, pageSize, pageNo, searchText = '') {
```

Comment:
Yes, verified it , it will be get the call in targeted solutions it will not affect there, so if there is an program id we are just adding it to match query for list api @VISHNUDAS-tunerlabs